src/static/containers/Home/index.js

```diff
@@ -54,5 +71,11 @@ const mapStateToProps = (state) => {
     };
 };
 
-export default connect(mapStateToProps)(HomeView);
+const mapDispatchToProps = (dispatch) => {
+    return {
+        actions: bindActionCreators(actionCreators, dispatch)
```

Comment:
I like your approach, but this means changing how this was implemented in the beginning. what do you think @lfrodrigues @pedrorodriguesgomes ?